cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ModelUtils.java

```diff
@@ -812,4 +815,60 @@ public static List<String> generateAllServiceDnsNames(String namespace, String s
 
         return dnsNames;
     }
+
+    /**
+     * Validate CPU requests and limits.
+     * 
+     * @param resources Compute resources configuration.
+     * @param owner Compute resources owner.
+     * 
+     * @return Error set.
+     */
+    public static Set<String> validateCpuResources(ResourceRequirements resources, String owner) {
+        return validateComputeResources(resources, "cpu", owner);
+    }
+
+    /**
+     * Validate memory requests and limits.
+     * 
+     * @param resources Compute resources configuration.
+     * @param owner Compute resources owner.
+     * 
+     * @return Error set.
+     */
+    public static Set<String> validateMemoryResources(ResourceRequirements resources, String owner) {
+        return validateComputeResources(resources, "memory", owner);
+    }
+
+    /**
+     * Validate compute resources requests and limits.
+     * 
+     * TODO: handle any type, not just cpu and memory.
```

Comment:
If you plan to do it, you should open an issue. TODO like this are from my experience useless ;-). Realistically, we do not have any real use case for this anyway right now I guess. So we can also just simply clarity that we handle only CPU and Memory here?